kolibri/plugins/learn/assets/src/actions.js

```diff
@@ -93,7 +93,7 @@ function _getCurrentChannelId() {
   return new Promise((resolve, reject) => {
     _getChannelList()
       .then((channelList) => {
-        if (channelList.length) {
+        if (channelList) {
```

Comment:
Problem was that channelList was null so it failed at checking the length of a null array.